controllers/module_controller.go

```diff
@@ -44,6 +44,13 @@ type moduleInstance struct {
 	tfClient TerraformCloudClient
 }
 
+var (
+	runCompleteStatus = map[string]struct{}{
```

Comment:
You could use the [tfe.RunStatus](https://pkg.go.dev/github.com/hashicorp/go-tfe#RunStatus) type here so you don't have to do the string type conversion [below](https://github.com/hashicorp/terraform-cloud-operator/pull/301/files#diff-49882eaa5202551554a16dcc2a3c75e7aa63405ce2002d668ef2bd48eb1e01bdR325).

Reply:
Thanks! Done.